driver/ppm_events_public.h

```diff
@@ -363,6 +363,11 @@ along with sysdig.  If not, see <http://www.gnu.org/licenses/>.
 
 #define PPM_PTRACE_IDX_MAX 2
 
+#define PPM_BPF_IDX_MAX 2
```

Comment:
I'd move this below `PPM_BPF_IDX_RES` because in case you'll add new IDX it will be easier to remember to update it IMO